src/components/PeopleFilters.tsx

```diff
@@ -27,66 +100,58 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {Array.from(
+              { length: centFilterLength },
+              (_, index) => String(index + firstCentury),
+            ).map(century => (
+              <Link
+                data-cy="century"
+                to={{
+                  search: getSearchWith({
+                    centuries: centuries.includes(century)
```

Comment:
DRY
move it to the variable
```suggestion
                    centuries: ...
```